Source/santad/SNTPolicyProcessor.h

```diff
@@ -60,4 +61,13 @@
 - (nonnull SNTCachedDecision *)decisionForFilePath:(nonnull NSString *)filePath
                                        identifiers:(nonnull SNTRuleIdentifiers *)identifiers;
 
+
+///
+/// Updates a decision for a given file and agent configuration.
+///
+/// Returns YES if the decision requires no futher processing NO otherwise.
+- (BOOL) decision:(nonnull SNTCachedDecision *) cd 
+          forRule:(nonnull SNTRule *) rule
+          withTransitiveRules:(BOOL) transitive;
```

Comment:
```suggestion
- (BOOL)decision:(nonnull SNTCachedDecision *) cd 
                forRule:(nonnull SNTRule *) rule
    withTransitiveRules:(BOOL) transitive;
```

Reply:
I'm not certain that clang-format will agree with the last line but there should be no space between the return type and method name